indico/htdocs/sass/modules/_surveys.scss

```diff
@@ -28,6 +28,17 @@
     padding: 1em;
 }
 
+%survey-questionnaire-handle {
+    @include icon-before('icon-grid2');
```

Comment:
i needed different classes in my javascript anyway. but yeah, could have used `js-whatever` classes and kept the same handle class for styling